src/Components/Tasks/Tasks.tsx

```diff
@@ -1,7 +1,75 @@
-import React from 'react'
+import { useContext, useEffect, useState } from "react";
+import { useNavigate } from "react-router";
+import { AuthContext } from "../../Context/AuthContext";
+import axios from "axios";
+import noData from '../../assets/noData.png';
+
 
 export default function Tasks() {
+  const [tasks, setTasks] = useState([]);
+
+  const { baseUrl,reqHeaders,role }: any = useContext(AuthContext);
+  const navigate = useNavigate();
+
+  const navigateToNew = () => {
+    navigate("/dashboard/add-task");
+  };
+
+  let getTasksList = () => {
```

Comment:
Using let to declare a variable that you will never reassign is not a good practice, use "const" instead